SPEC.md

```diff
@@ -314,22 +316,22 @@ Retrievable at `http://$AC_METADATA_URL/acMetadata/v1/container`
 
 | Entry       | Description |
 |-------------|-------------|
-|annotations/ | Top level annotations from container runtime manifest.|
-|manifest     | The container manifest JSON |
-|uid          | The unique execution container uid.|
+|annotations/ | Top level annotations from container runtime manifest. |
+|manifest     | The container manifest JSON. |
+|uid          | The unique execution container uid. |
 
 ### App Metadata
 
 Every running process will be able to introspect its App Name via the `AC_APP_NAME` environment variable.
 This is necessary to query for the correct endpoint metadata.
 
-Retrievable at `http://$AC_METADATA_URL/acMetadata/v1/apps/${ac_app_name}/`
+Retrievable at `http://$AC_METADATA_URL/acMetadata/v1/apps/$AC_APP_NAME/`
 
 | Entry         | Description |
 |---------------|-------------|
-|annotations/   | Annotations from image manifest merged with app annotations from container runtime manifest.|
+|annotations/   | Annotations from image manifest merged with app annotations from container runtime manifest. |
 |image/manifest | The original manifest file of the app. |
-|image/id       | Cryptographic image ID this app is on.|
+|image/id       | Cryptographic image ID this app is on. |
```

Comment:
I don't know why the word "cryptographic" is in here because I think that makes it slightly confusing. This is the image ID described earlier:
> An image is addressed and verified against the hash of its uncompressed tar file, the image ID. The default digest format is sha512, but all hash IDs in this format are prefixed by the algorithm used (e.g. sha512-a83...).